templates/ai/content/ai-basics/index-content.html

```diff
@@ -1,16 +1,16 @@
-<h2 id="the-quick--dirty-intro-to-ai">The quick &amp; dirty intro to AI</h2>
-<h4 id="what-is-ai-in-2023">What is AI in 2023?</h4>
```

Comment:
these ids are needed for sidebar scroll

Reply:
Don't know how to fix this with your build script.

Reply:
The build script is not generating the ids properly for me.